packages/payment-widget/src/lib/payment-widget.svelte

```diff
@@ -193,7 +195,7 @@
 
   <section class="rn-payment-widget-body">
     <h2>Pay with crypto</h2>
-    <Button
+    <button
```

Comment:
@aimensahnoun for future reference, odd changes like this warrant a comment in the code to explain why we're breaking from convention.